doc/user/data/sql_funcs.yml

```diff
@@ -507,55 +507,55 @@
   functions:
   - signature: 'jsonb_array_elements(j: jsonb) -> Col<jsonb>'
     description: "`j`'s elements if `j` is an array."
-    url: "/docs/sql/types/jsonb/#jsonb_array_elements"
+    url: "/sql/types/jsonb/#jsonb_array_elements"
```

Comment:
I think there's a missing `| relURL` filter somewhere, based on the lint failure.